app/datatables/sample_datatable.rb

```diff
@@ -110,13 +110,13 @@ def data
         '1': assigned_cell(record),
         '2': record.name,
         '3': record.sample_type.nil? ? I18n.t('samples.table.no_type') : record.sample_type.name,
-        '4': record.sample_group.nil? ?
-        "<span class='glyphicon glyphicon-asterisk'></span> " + I18n.t("samples.table.no_group") :
-        "<span class='glyphicon glyphicon-asterisk' style='color: #{record.sample_group.color}'></span> " + record.sample_group.name,
+        '4': sample_group_cell(record),
         "5": I18n.l(record.created_at, format: :full),
-          "6": record.user.full_name,
-          "sampleInfoUrl": Rails.application.routes.url_helpers.edit_sample_path(record.id),
-          "sampleUpdateUrl": Rails.application.routes.url_helpers.sample_path(record.id)
+        "6": record.user.full_name,
+        "sampleInfoUrl":
```

Comment:
Can you change double quotes to single quotes here to be consistent with other lines?